lwdcalculator.py

```diff
@@ -3,8 +3,16 @@ class Calculator:
     result : str
 
     def get_input(self):
-        ''' get input '''
-        self.user_response = input("Please enter your calculation : ")
+      ''' get input '''
+      values = input("Please enter two numbers : ")
+
+      while isinstance(values, str):
+          try:
+            values = [int(i) for i in values.split()]
+          except ValueError:
+            values = input("Input should contain numbers only, please try again : ")                    
+      self.user_response = values
+      return values
```

Comment:
I would say this is one way to process the string input from the user.
But we wouldn't want this in the **get_input** method.
**get_input** method should be responsible for getting the values in string, hence **user_response**.
later, **user_response** will be processed differently in the respective methods.
So maybe you could do all this, but in a new calculate method.